tasks/task449_app_review_classification.json

```diff
@@ -0,0 +1,5945 @@
+{
+    "Contributors": [
+        "Vijay Kumawat"
+    ],
+    "Source": [
+        "app_reviews (dataset link- https://huggingface.co/datasets/app_reviews)"
+    ],
+    "Categories": [
+        "Classification"
+    ],
+    "instruction_language": "English",
+    "Definition": "In this task, you are given a review of an app. Your task is to classify as \"Bad\",\"Very Bad\", \"Excellent\"\"Good\"and \"Average\" on the basis of the star rating.",
+    "Positive Examples": [
+        {
+            "input": "Good job Tried a few . This has to be the most stable and best to my liking . And a whole world of settings to play with. However we are quite limited on frequency band width id like to see it lowered to 14.mhz if possible",
+            "output": "Excellent",
+            "explanation": "This particular review has a rating of 5 which means that the app falls in in Excellent category for the reviewer."
```

Comment:
Task Definition : In this task, you are given a review of an app. Your task is to classify as \"Bad\",\"Very Bad\", \"Excellent\"\"Good\"and \"Average\" on the basis of the **star rating**.

But when you give the input to the model you are not providing the start rating. Then how would the model follow this instruction to determine the review quality? The explanation should mention why this is excellent. 
@swarooprm How would these student write the explanations in such a case. There seems to be many such dataset

Reply:
yes, there should be no mention of star rating since such data is not provided in input